.github/workflows/daily-build-renew-cache.yml

```diff
@@ -0,0 +1,36 @@
+name: Daily Build Renew Cache
+on:
+  schedule:
+    - cron: '0 7 * * *'
+
+jobs:
+  docker:
+    name: Build Docker with Cache
+    runs-on: ubuntu-20.04
+    environment: production
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Determine Meta
+        run: echo "REF=${GITHUB_REF##*/}" >> $GITHUB_ENV
```

Comment:
How does this work?

Reply:
In the github environment, `GITHUB_REF` is set to like  `refs/tags/0.0.001`
This sets `REF=0.0.001` so we know what to tag it later?